src/Rector/v7/v4/DropAdditionalPaletteRector.php

```diff
@@ -129,10 +130,14 @@ public function refactor(Node $node): ?Node
                     continue;
                 }
                 $typeItem->value = new String_(implode(',', $newFieldStrings));
+                $hasAstBeenChanged = true;
             }
         }
 
-        return $node;
+        if ($hasAstBeenChanged) {
```

Comment:
Could you please change this to `return $hasAstBeenChanged ? $node : null;` to be consistent.

Reply:
of course. Just realized while going through the files that a ternary would be nice and didn't go back to update the rest